src/demo/react/DemoButton.jsx

```diff
@@ -0,0 +1,24 @@
+import React, { createElement } from 'react';
+import '../../components/10-atoms/button';
+import createAXAButtonReact from '../../components/10-atoms/button/index.react';
+
+const AXAButtonReact = createAXAButtonReact(createElement);
+
+const DemoButton = () => {
+  const handleAXAButtonClick = () => {
+    /* eslint-disable no-alert */
+    window.alert('Oo yeah you clicked me');
```

Comment:
Can't we workaround this ugly eslint rule somehow? Like can we use something else than alert or completely remove this rule from eslint (or ignore ".story.js" files)?

Reply:
#6e36fa2

Reply:
6e36fa2622bcb1a6cda401d9535aee5d8ec02249

Reply:
6e36fa2

Reply:
Improve example without a alert see 6e36fa2